zk/groth16/src/serde.rs

```diff
@@ -0,0 +1,23 @@
+pub mod serde_fr {
+    use ark_bn254::Fr;
+    use ark_ff::biginteger::BigInteger as _;
+    use num_bigint::BigUint;
+    use serde::{Deserialize as _, Deserializer, Serializer};
+
+    pub fn serialize<S>(item: &Fr, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let hex = hex::encode(item.0.to_bytes_le()); // Convert `Fr` to hex representation
```

Comment:
we should use `into_bigInt` as in the docs

Reply:
Where is that in the docs? You mean it should be a `str` with the bignum representation?

Reply:
where do you see the internal BigInt in the public docs? If I look at the source code it's `#[doc(hidden)]` with a comment to use the `into_bigint` https://docs.rs/ark-ff/0.5.0/src/ark_ff/fields/models/fp/mod.rs.html#106-112

Reply:
Ah, ok, i can try that. Thanks!